dotnet/src/InternalUtilities/src/Schema/KernelJsonSchemaBuilder.cs

```diff
@@ -24,13 +24,7 @@ namespace Microsoft.SemanticKernel;
 internal static class KernelJsonSchemaBuilder
 {
     private static JsonSerializerOptions? s_options;
-    internal static readonly AIJsonSchemaCreateOptions s_schemaOptions = new()
-    {
-        IncludeSchemaKeyword = false,
```

Comment:
These settings all match the current defaults, so I removed them altogether.

Reply:
Do we need the s_schemaOptions instance at all then?